#Reading variables
string id_reading
string id_mission
string id_section
string antenna_port
string timestamp
string rf_phase
string rssi
string freq
string mux1
string mux2
string epc
string device_id
string robot_pose